apps/docs/content/pages/getting-started.mdx

```diff
@@ -0,0 +1,187 @@
+
+## Detailed Installation
+
+### Components
+
+Hopper Components are React Comtponents, uf you are using another framework, or React Native, you should use the [components tokens](/tokens/getting-started/introduction) directly when developping your own components.
+
+#### Installation
+
+Run one of the following commands in your terminal to install the Hopper Components package.
+
+<PackageInstallation library="components" mod="prod" />
+
+#### Usage
+
+A Provider is required to use Hopper Components. Wrap your application with the HopperProvider component to use our components.
+
+```jsx
+import { Button } from "@hopper-ui/components";
+
+<HopperProvider theme="dark">
+    <Button>Hello World</Button>
+</HopperProvider>
+```
+
+### Tokens
+
+Hopper Tokens are a collection of design tokens that should be used to style your application. [Semantic tokens](#) are available for everyone, where [components tokens](#) are designed with non React users in mind.
+
+#### Installation
+
+<PackageInstallation library="tokens" mod="prod" />
+
+#### Usage
+
+##### CSS
+
+Import the tokens in your project by doing the following:
+
+```css
+@import "@hopper-ui/tokens/tokens.css";
+```
+
+If your application supports a dark mode, import dark mode tokens as well:
+
+```css
+@import "@hopper-ui/tokens/dark/tokens.css";
+```
+
+You can now use Hopper tokens in your project by using the following syntax:
+
+```css
+.my-class {
+    background-color: var(--hop-primary-surface-weak);
+}
+```
+
+##### Fonts
+
+In order to use Hopper fonts in your project, import the following:
+
+```css
+@import "@hopper-ui/tokens/fonts.css";
+```
+
+All the necessary font-face declarations are now imported in your project.
+
+### Icons
+
+#### Installation
+
+<PackageInstallation library="icons" mod="prod" />
+
+#### Usage
+
+You can now use [icons](/icons/react-icons/icon-library) and [rich icons](/icons/react-icons/rich-icon-library) in your project by using the following syntax:
+
+```tsx
+import { AddIcon } from "@hopper-ui/icons";
+import { ConversationRichIcon } from "@hopper-ui/icons";
+
+<AddIcon />
+<ConversationRichIcon />
+```
+
+#### Non-standard installations
+
+##### SVG Icons
+
+If your project is not using the React framework, you can use the SVG icons directly. You can find the SVG icons in the `@hopper-ui/svg-icons` package.
+
+<PackageInstallation library="svg-icons" mod="prod" />
+
+You can import the SVG directly into your JavaScript file. Keep in mind that if you plan to use SVG icons as components, you'll need to ensure that your bundler is configured to handle SVG parsing appropriately.
+
+```tsx
+import alertIcon from "@hopper-ui/svg-icons/alert-24.svg";
+```
+
+Alternatively, you can utilize SVG icons within your CSS in the following manner:
+
+```css
+.myComponent {
+    background-image: url("@hopper-ui/svg-icons/alert-24.svg");
+}
+```
+
+##### React Icons Standalone Installation
```

Comment:
I'd remove React Icons because it's redundant 

```suggestion
##### Standalone Installation
```

And I don't quite understand the purpose of this section, isn't it a light version of using Hopper?

Reply:
@alexasselin008 can you confirm this?

Reply:
if it can simplify this page we can remove it. But there needs to be orbiter or hopper components installed with the HopperPRovider/ThemePRovider for the icons to work properly